client.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/http/cookiejar"
+)
+
+const (
+	baseAuthTermURL = "https://ssb.cc.nd.edu/StudentRegistrationSsb/ssb/term/search?mode=search&term="
+)
+
+func setupClient() (*http.Client, error) {
+	cookieJar, err := cookiejar.New(nil)
+
+	if err != nil {
+		log.Fatal(err)
+		return nil, err
+	}
+
+	client := &http.Client{
+		Jar: cookieJar,
+	}
+
+	return client, err
+}
+
+func authenticateClient(c *http.Client, term string) {
+	authURL := baseAuthTermURL + term
+	fmt.Println(authURL)
```

Comment:
always prefer `log.Xxx` to `fmt.Xxx`: in this case use `log.Print(authURL)` (no need to add `\n` to it, the logger does it automatically if there isn't one)